SCClassLibrary/Common/Collections/Dictionary.sc

```diff
@@ -493,6 +512,21 @@ IdentityDictionary : Dictionary {
 		parent = newParent;
 	}
 
+	== { arg that;
+		^this.superPerform('==', that)
+		and: { parent == that.parent }
+		and: { proto == that.proto }
+		and: { know == that.know }
+	}
+
+	hash {
+		var hash = hash bitXor: know.hash;
```

Comment:
sorry if i wrote this in my last review... should just start `var hash = know.hash`, we can't xor against nil